packages/n4s/src/enforce/compounds/anyOf.js

```diff
@@ -0,0 +1,3 @@
+export default function anyOf(value, ...rules) {
+  return Boolean(!rules) || rules.some(fn => fn(value));
```

Comment:
Not sure what you were trying to achieve with the first condition - rules will always be an array, hence, always truthy, so that condition will always fail. Maybe you meant to check `rules.length`? Anyway, in our case, since you went with `fail` for no rules, your current implementation should just work.
```suggestion
  return rules.some(fn => fn(value));
```